modules/dists/BlockDist.chpl

```diff
@@ -673,6 +673,12 @@ proc Block.getChunk(inds, locid) {
   //
   // TODO: Does using David's detupling trick work here?
   //
+  // Vass 2023-03: the chunk should really be computed as:
+  //   const chunk = inds[locDist(locid).myChunk];
+  // because we are looking for a subset of 'inds'.
+  // I did not make this change because it slightly bumps comm counts in:
+  //   distributions/robust/arithmetic/performance/multilocale/assignReindex
+  //
   const chunk = locDist(locid).myChunk((...inds.getIndices()));
```

Comment:
Yes, domains have strides. If the domain is "!stridable", then the strides in all dimensions are 1. Otherwise each dimension has its own stride.

The behavior of the existing code is: `chunk` ends up with a positive stride with the old rule. This is because `locDist(locid).myChunk` is the portion of the global index space that resides on the locale 'locid', so it has positive stride(s) and that is not reversed under the old rule. Under the new rule, `chunk` ends up with a negative stride when `inds` has a negative stride. This is desired because what we are doing is partitioning `inds` across locales. Which has no business reversing the direction.

However, apparently whether the direction is reversed does not matter for the correctness of the code. Therefore I am OK leaving this code as-is.